databuilder/extractor/dashboard/mode_dashboard_extractor.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+from requests.auth import HTTPBasicAuth
+from typing import Any  # noqa: F401
+
+from databuilder import Scoped
+from databuilder.extractor.base_extractor import Extractor
+from databuilder.extractor.restapi.rest_api_extractor import RestAPIExtractor, REST_API_QUERY, MODEL_CLASS
+from databuilder.rest_api.base_rest_api_query import RestApiQuerySeed
+from databuilder.rest_api.rest_api_query import RestApiQuery
+
+# CONFIG KEYS
+ORGANIZATION = 'organization'
+MODE_ACCESS_TOKEN = 'mode_user_token'
+MODE_PASSWORD_TOKEN = 'mode_password_token'
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ModeDashboardExtractor(Extractor):
+    """
+    A Extractor that extracts core metadata on Mode dashboard.
+    It extracts list of reports that consists of:
+        Dashboard group name (Space name)
+        Dashboard group id (Space token)
+        Dashboard description (Space description)
```

Comment:
is it dashboard group description or dashboard description? It seems to be the former as you mentioned  `Space description`...

Reply:
Good catch. Will update.